src/ray/gcs/format/gcs.fbs

```diff
@@ -4,6 +4,15 @@ enum Language:int {
   JAVA = 2
 }
 
+// The channel that Add operations to the Table should be published on, if any.
+enum TablePubsub:int {
```

Comment:
Call this TableType? It is probably more generally useful than for pub/sub.

Reply:
@pcmoritz and I decided to keep this as a pubsub-specific type and to introduce a string enum in a future PR to prefix each table entry's key.